test/it/client-listUsers.js

```diff
@@ -51,7 +67,7 @@ describe('client.listUsers().each()', () => {
   it('should return User models', () => {
     return client.listUsers().each(user => {
```

Comment:
Question for @vijetmahabaleshwar-okta or @lboyette-okta - do these tests also make calls that would need to be handled by the test-server (returning taped responses)?

My implementation of async collections is different, and wouldn't require tests for each(), iteration abort, etc. If my tests **didn't** make these additional repeated calls to `/api/v1/users`, would the test-server think I didn't finish the scenario?